views/default/page/layouts/elements/sidebar.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/**
+ * Layout sidebar
+ * 
+ * @uses $vars['sidebar'] Sidebar view
+ */
+
+$sidebar = elgg_extract('sidebar', $vars);
+if ($sidebar === false) {
+	return;
+}
+?>
+<div class="elgg-sidebar elgg-layout-sidebar clearfix">
+	<?= elgg_view('page/elements/sidebar', $vars) ?>
+</div>
```

Comment:
Curious, why not just add the wrapper to page/elements/sidebar? A consistency thing?